doc/nrf/releases/release-notes-latest.rst

```diff
@@ -102,4 +102,11 @@ The following list summarizes the most important changes inherited from the upst
 Documentation
 =============
 
-There are no entries for this section yet.
+In addition to documentation related to the changes listed above, the following documentation has been updated:
+
+Applications and samples
+------------------------
+
+* Bluetooth:
+
+  * :ref:`ble_samples` - Changed bluetooth  sample name Central DFU SMP to :ref:`Central SMP Client <bluetooth_central_dfu_smp>`.
```

Comment:
```suggestion
  * :ref:`ble_samples` - Changed the Bluetooth sample Central DFU SMP name to :ref:`Central SMP Client <bluetooth_central_dfu_smp>`.
```

Reply:
And please move this back to the location you had originally.

Reply:
Done.